src/main/java/soconnect/logic/commands/SearchCommand.java

```diff
@@ -44,7 +45,11 @@ public SearchCommand(Predicate<Person> predicate, Predicate<Person> alternativeP
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        model.updateFilteredPersonList(predicate);
+        try {
+            model.updateFilteredPersonList(predicate);
+        } catch (Exception e) {
```

Comment:
Are you sure it is a good idea to catch `exception`? Is there anything more specific?

Reply:
No, it's not a good idea haha. I am just honestly paranoid that some exceptions I don't know are not handled well. But yes, this is not good too. After some screening, I managed to reduce it to null and invalid argument exception, though let's hope that indeed no exception is not handled and this is just a paranoia.